graspologic/plot/plot.py

```diff
@@ -732,7 +957,7 @@ def degreeplot(
     ----------
     X : np.ndarray (2D)
         input graph
-    labels : 1d np.ndarray or list, same length as dimensions of ``X``
+    labels : 1d np.ndarray or list, same length as dimensions of X
```

Comment:
same comment here, why? in general if you are trying to make improvements to other things not related to PR that is great but it gets confusing when they are packaged together, so a separate PR will usually be easier. though here I'm not sure why this change is desired.

Reply:
I really am unsure... i really did not actively change this.. and i cannot find the commit where this happened..